src/common/input-map/input-map.scss

```diff
@@ -2,18 +2,18 @@
     @extend .list-unstyled;
     >*{
         margin-bottom: $space-size;
-        padding: $space-size/2;
     }
     >*:nth-child(2n){
         background-color: $gray-lighter;
     }
 }
 .input-map .form-group{
     padding-left: 0px;
+    margin-bottom: 0.5em;
     >*{
         display: inline-block;
-    }
-    label, input{
-        width: $sidebar-width;
+        &:last-child{
+            margin-left: 15px;
```

Comment:
@afijal Use `$space-size` which in fact represents 15px instead of using magic numbers